utilities/api/python_client_semgrep_api.py

```diff
@@ -63,16 +64,20 @@ def get_all_findings(projects, headers):
     """
     for project in projects['projects']:
         project_name = project['name']
+        primary_branch = project['default_branch']
```

Comment:
```suggestion
        primary_branch = project['primary_branch']
```
The dedicated `primary_branch` attribute is what we should be using (see [doc](https://semgrep.dev/api/v1/docs/#tag/Project/operation/semgrep_app.saas.handlers.repository.openapi_list_recent_projects)). The default branch is just the SCM default and doesn't reflect if there have been overrides in Semgrep.

Reply:
Oh good catch!